javascript/javascript1/week1/ageCalculatorApp.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+ <script>
+
+     //Age-fying; a future age calculator
+let yearOfBirth = 1989;
```

Comment:
Always use ‘const’ unless you need to reassign the variable. You should very rarely be using let

Reply:
I will replace all lets with const, Thanks